broker-daemon/broker-rpc/orderbook-service/get-orderbook.js

```diff
@@ -36,6 +49,12 @@ async function getOrderbook ({ params, logger, orderbooks }, { GetOrderbookRespo
       }
     })
 
+    // Bids are sorted in descending order so the best bid (i.e. highest bid) is returned first
+    bids.sort((a, b) => parseFloat(b.price) - parseFloat(a.price)).splice(limit)
```

Comment:
So I think we ideally want to the limit the number of records we get from the orderbook instead of getting all the records from the orderbook and then just returning a certain amount to the CLI. 

We have a method called `streamOrdersAtPriceOrBetter` which is defined on our PriceIndex. I think what you will want to do is something similar to what we have done in `orderbook.getBestOrders`, but instead of limited by depth, you will be limited by number of records. So your code here will look something like:

const bids = `orderbook.getOrders(bid, limit)`
const asks = `orderbook.getOrders(ask, limit)`

We could actually remove the iteration above: https://github.com/sparkswap/broker/pull/381/files#diff-7eb805b598ec65392196d352232266ffR43 and if we don't want to limit the number of records, just not pass in a limit.

And then `orderbook.getOrders` would look similar to `orderbook.getBestOrders` but you would use the limit to finish the stream when you got to the provided limit.